src/devices/vdml_link.c

```diff
@@ -34,36 +34,32 @@ static uint32_t _clear_rx_buf(v5_smart_device_s_t* device) {
     vexDeviceGenericRadioReceiveAvail(device->device_info));
 }
 
-// internal wrapper for initialization
-static uint32_t _link_init_wrapper(uint8_t port, const char* link_id, link_type_e_t type, bool ov) {
-    // claim_port_i(port - 1, E_DEVICE_RADIO); is not used because it's a generic serial device 
-    // the reason behind why it can be a radio or a serial device is because
-    // a radio is not the highest port radio, it will not be registered with vexos.
-	if (!VALIDATE_PORT_NO(port - 1)) {
-		errno = EINVAL;
-		return PROS_ERR;
-	}
-    // by default, the vexlink radio in the lower port should be E_DEVICE_NONE
-    // or, the only radio on the brain
-    v5_device_e_t plugged_device = registry_get_plugged_type(port - 1);
-    if(plugged_device != E_DEVICE_NONE && plugged_device != E_DEVICE_RADIO) {
+uint32_t link_init(uint8_t port, const char* link_id, link_type_e_t type) {
+    v5_device_e_t plugged_device = registry_get_plugged_type(port);
+    if(plugged_device != E_DEVICE_RADIO) {
+        if(plugged_device == E_DEVICE_SERIAL) {
+            kprintf("PROS does not currently support the use of multiple radios.");
+        }
```

Comment:
Also if anything, this check should be for E_DEVICE_NONE since lower radio ports show up as "no device", but again I'm not sure if it's necessary at all tbh.

Reply:
Yeah, I think it makes sense to note the support for only one radio in the documentation then, also I forgor what the radio is registered as/plugged type is when there is only one radio